systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"net/http"
+	"testing"
+
+	e2eCommon "github.com/dgraph-io/dgraph/graphql/e2e/common"
+	utilsCommon "github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+var (
+	headerAlpha1 = http.Header{}
+	headerAlpha2 = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	restoreLocation = "/data/backups/backupTemp/"
+	backupDst       = "/data/backups/backupTemp/"
+	localBackupDest = "./data/backups"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	defer utilsCommon.DirCleanup(t)
```

Comment:
why are we only doing cleanup? Where do we create the dir that we are cleaning up here?

Reply:
backupTemp folder gets created when the code takes a backup(backupdest is /data/backups/backupTemp).

Now the test is using shared volume and I have removed the DirCleanup() call from the test.